src/harness/sas_testcase.py

```diff
@@ -96,13 +96,42 @@ def assertRegisteredAndGranted(self, registration_request, grant_request,
       self.assertEqual(grant_resp['response']['responseCode'], 0)
 
       list_of_cbsd_ids_and_grant_ids.append({
-          'cbsdId': grant_resp['cbsdId'],
-          'grantId': grant_resp['grantId']
+        'cbsdId': grant_resp['cbsdId'],
+        'grantId': grant_resp['grantId']
       })
 
     # Return cbsd_ids and grant_ids
     return list_of_cbsd_ids_and_grant_ids
 
+  def assertHeartbeatsSuccessful(self, cbsd_ids, grant_ids, operation_states):
```

Comment:
I think option 1 is fine. But with option 1 the input will be `assertHeartbeatsSuccessful(self, tuple_of_cbsd_ids_grant_ids, operation_states):
  cbsd_ids, grant_ids = tuple_of_cbsd_ids_grant_ids` ?